bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -942,19 +942,24 @@ void createDisplay (DeviceData data) {
 }
 
 static long create32bitDIB (Image image) {
+	return create32bitDIB (image, DPIUtil.getDeviceZoom());
+}
+
+static long create32bitDIB (Image image, int zoomLevel) {
```

Comment:
Can you rename zoomLevel into zoom for consistency?

Reply:
Done